source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,204 @@
+
+## A publisher publishes content on GOV.UK
+
+GOV.UK has the following applications to publish content:
+
+- Mainstream Publisher
+- Whitehall
+- Content Publisher
+- Specialist Publisher
+- Travel Advice Publisher
+- Collections Publisher
+
+The following content states what happens when a user publishes content to live and to draft using Mainstream Publisher ("Publisher").
+
+### A user publishes content to live
+
+A user signs into their Signon account. The user creates or changes content in Publisher and saves that content in draft.
+
+Publisher makes a request to Publishing API to create or update the appropriate draft content.
+
+When the user is ready to publish the content, they press __Publish__ in the Publisher which sends a request to Publishing API.
+
+The Publishing API then updates the Content Store with the new or updated content.
+
+Content Store updates Router to say there is now content at a particular URL and says which frontend application will render that content.
+
+The Publishing API tells the Search API that the content is live.
+
+Search API updates its indexes so the content can be searched for by users.
+
+### A user publishes content to draft
+
+A user signs into their Signon account. The user creates or changes content in Publisher and saves that content in draft.
+
+Publisher makes a request to Publishing API to create or update the appropriate draft content.
+
+Before publishing content to live, a user can preview this draft content. To enable this, the following applications have both draft and live data and environments:
+
+- Router
+- Content Store
+- Government Frontend
+
+The Publishing API updates the draft Content Store when a user updates content, and then updates the live Content Store when the user then publishes the content.
+
+Access to the draft environment is restricted. The Authenticating Proxy sits in front of the draft Router, and checks whether a user should have access. The Proxy does this either through Signon credentials, or through a secret token if it is for fact checkers.
+
+### The publishing message queue
+
+When a user publishes content, the Publishing API tells the Search API that the content is live. The Publishing API does this by writing to the publishing message queue. The Search API is subscribed to this message queue and responds when the Publishing API writes to the queue. The following applications also subscribe to the publishing message queue.
+
+#### Cache Clearing Service
+
+When a user publishes new or updated content, the Cache Clearing Service tells Fastly to clear its cache for that page, meaning people can access the new content faster.
+
+#### Email Alert Service
+
+When a user publishes new or updated content, this application emails subscribers to tell them that new or updated content has been published.
+
+#### Content Data API
+
+When a user publishes new or updated content, this application updates its data on what pages are live on GOV.UK.
+
+## Definitions
```

Comment:
A `Definitions` section is a nice idea, but given that most definitions are only a sentence or two, I wonder if we can avoid the need for this section by inlining a bit more of this content into the document?

Reply:
have given it a try, see what you think